src/main/kotlin/com/hj/leetcode/kotlin/problem646/Solution.kt

```diff
@@ -0,0 +1,23 @@
+package com.hj.leetcode.kotlin.problem646
+
+/**
+ * LeetCode page: [646. Maximum Length of Pair Chain](https://leetcode.com/problems/maximum-length-of-pair-chain/);
+ */
+class Solution {
+    /* Complexity:
+     * Time O(NLogN) and Space O(N) where N is the size of pairs;
+     */
+    fun findLongestChain(pairs: Array<IntArray>): Int {
+        val sortedPairs = pairs.sortedBy { it[1] }
+        var result = 0
+        var tailSecond = Int.MIN_VALUE
```

Comment:
rename to `chainTailSecond` which is clearer

Reply:
Resolved by https://github.com/hj-core/hj-leetcode-kotlin/pull/133/commits/aa37e24bf05b48ddb859924a73a5b43a5941517a